CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Changelog
 
+## [1.19.2](https://github.com/brave/brave-ios/releases/tag/v1.19.2)
+
+ - Implemented URL scheme handlers for both "http" and "https". ([#2784](https://github.com/brave/brave-ios/issues/2784))
```

Comment:
nit: believe it should be HTTP & HTTPS (capitalized).
* Example: https://en.wikipedia.org/wiki/HTTPS